backend/tarsy/config/settings.py

```
@@ -244,11 +244,19 @@ def get_template_default(self, var_name: str) -> Optional[str]:
+        logger = get_module_logger(__name__)
+        
         # Convert template variable name to settings attribute name
-        # KUBECONFIG -> kubeconfig_default
-        # PROMETHEUS_URL -> prometheus_url_default
+        # VARNAME -> varname_default
+        # VAR_NAME -> var_name_default
         default_attr = f"{var_name.lower()}_default"
-        return getattr(self, default_attr, None)
+        default_value = getattr(self, default_attr, None)
+        
+        logger.debug(f"Template default lookup for '{var_name}': attribute='{default_attr}', value='{default_value}'")
```

Ruff (0.12.2)

Line too long (118 > 88)

(E501)